src/sanitize-html.js

```diff
@@ -0,0 +1,11 @@
+// Sanitize HTML and prevent XSS attacks using DOMPurify
+// eg. DOMPurify.sanitize('<img src=x onerror=alert(1)//>'); // becomes <img src="x">
+
+const createDOMPurify = require('dompurify');
+const { JSDOM } = require('jsdom');
+
+module.exports.run = async function(text) {
+  const { window } = new JSDOM('');
+  const DOMPurify = createDOMPurify(window);
+  return Promise.resolve(DOMPurify.sanitize(text));
```

Comment:
I'm curious about this, why wrapping it in a promise? I checked the [dompurify repo](https://github.com/cure53/DOMPurify) and it doesn't seem to need that.

Reply:
Some DOM implementations will be sync, others async.  If we wrap this in a Promise based API, we can choose either or, without changing all the call sites.